sdk/communication/azure-communication-messages/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.2.0b1 (2025-01-15-preview)
+## 1.2.0b1 (2025-01-15)
```

Comment:
I thought release date should be 2025-02-11 since we are planning to release today

Reply:
I have used 2025-01-15 as release date for Rest Api pr and SDK , hence would to be consistent.